server/middleware/levels.coffee

```diff
@@ -71,7 +101,7 @@ module.exports =
       unless course.get('free') or req.user.isEnrolled()
         throw new errors.PaymentRequired('You must be enrolled to access this content')
         
-      lang = classroomWithLevel.get('aceConfig')?.language
+      lang = targetLevel.primerLanguage or classroomWithLevel.get('aceConfig')?.language
```

Comment:
No, because both properties are set previously and at the same time, and there's a check a few lines earlier to make sure classroomWithLevel has been set and to through a PaymentRequired error otherwise.

```
unless classroomWithLevel
  throw new errors.PaymentRequired('You must be in a course which includes this level to play it')
```